### ISSUE TITLE
call for key protocol ideas

### ISSUE THREAD

**Issue by rosee8** [ethdevberlin/OIPS#38]
Hi smart people and blockchain developer

Maybe you know our key protocol. it is quite… let’s say old school. 

What about creating something cool there? Do you think it is possible to use some great technology and invent something to improve the key protocol? Something cooler than digitalizing it into a google spreadsheet. Something creative. ✨ 🦄

What do you think?

**Comment by kuzdogan** [comment]
Key protocol == Office leave protocol?

**Comment by ligi** [comment]
I think it is about handing out physical keys - and I suggest replacing it by #33

**Comment by rosee8** [comment]
yes, what Ligi says, about handing out physical keys.
The automated lock is a great idea. It just seems it's not easy and fast to realize.
And i think we would keep the physical keys (at least some) anyways.

**Comment by ligi** [comment]
Yes we can/should keep some physical keys - "replace" was the wrong word here. "augment" might be a better word.
